python/tvm/script/builder/tir/prim_func_frame.py

```diff
@@ -40,3 +41,77 @@ def arg(name, obj) -> Union[Var, Buffer]:
 
 
 setattr(prim_func, "dispatch_token", "tir")
+
+
+def func_attr(attrs) -> DictAttrs:
+    return _ffi_api.FuncAttrs(
+        make_node("DictAttrs", **attrs)
+    )  # pylint: disable=no-member # type: ignore
+
+
+def ret(ret_type) -> Union[Var, Buffer]:
+    return _ffi_api.Ret(ret_type)  # pylint: disable=no-member # type: ignore
```

Comment:
let's find a better name given it conflicts with an intrinsic: https://github.com/junrushao1994/tvm/blob/0df69611b2fb46724a0023dd8d389c9a1ecedcb8/include/tvm/tir/op.h#L89

how about:


```suggestion
def func_ret(ret_type) -> Union[Var, Buffer]:
    return _ffi_api.FuncRet(ret_type)  # pylint: disable=no-member # type: ignore
```